komand/server.py

```diff
@@ -98,13 +120,53 @@ def handler(prefix, name, test):
                 r.status_code = status_code
                 return r
 
+        @app.route("/api_spec")
+        def api_spec():
+            """API spec details endpoint.
+            ---
+            get:
+              description: Get API spec details
+              responses:
+                200:
+                  content:
+                    application/json:
+                      type: object
+            """
+
+            return self.spec.to_yaml()
+
+        @app.route("/info")
+        def plugin_spec():
+            """Plugin spec details endpoint.
+            ---
+            get:
+              description: Get plugin details
+              responses:
+                200:
+                  content:
+                    application/json:
+                      schema: PluginSchema
+            """
+
+            response = {
+                "name": self.plugin.name,
+                "description": self.plugin.description,
+                "version": self.plugin.version,
+                "vendor": self.plugin.vendor,
+                "number_of_workers": self.workers,
+                "threads": self.threads
+            }
+            return jsonify(PluginSchema().dump(response))
+
         return app
 
     def start(self):
         """ start server """
         with self.app.app_context():
             try:
                 arbiter = Arbiter(self)
+                self.spec.path(view=self.app.view_functions['api_spec'])
+                self.spec.path(view=self.app.view_functions['plugin_spec'])
```

Comment:
```suggestion
                self.spec.path(view=self.app.view_functions['plugin_info'])
```

To match Zac's change